explorer_frontend/.env

```diff
@@ -1,2 +1,4 @@
 VITE_APP_TITLE="=nil; zkSharding"
 VITE_APP_DESCRIPTION="zkSharding for Ethereum"
+VITE_GET_ENDPOINT_URL=https://t.me/NilDevnetTokenBot
```

Comment:
use runtime config instead of env. look at `public/runtime-config.toml` and how it is resolved with typescript support in `src/features/runtime-config`

Reply:
Sure, should we then remove .env completely?

Reply:
we need this for title and description because they are used in index.html as well.